core/src/main/java/io/grpc/internal/CensusTracingModule.java

```diff
@@ -60,7 +60,16 @@
 final class CensusTracingModule {
   private static final Logger logger = Logger.getLogger(CensusTracingModule.class.getName());
 
-  @Nullable private static final AtomicIntegerFieldUpdater<ClientCallTracer> callEndedUpdater;
+  @VisibleForTesting
+  static final Attributes.Key<SpanId> SPAN_ID_ATTRIBUTES_KEY = Attributes.Key
+      .create("census-span-id");
+
+  @VisibleForTesting
+  static final Attributes.Key<TraceId> TRACE_ID_ATTRIBUTES_KEY = Attributes.Key
+      .create("census-trace-id");
```

Comment:
We need to expose those keys to our internal framework. If we had moved Census integration to a separate artifact, we would simply expose those keys from a public class in that artifact. For now, we have to keep it internal. Maybe we can add a public `CensusInternalAccessor` class that exposes these keys.

Reply:
`io.grpc` package does not have Census dependency, where should this accessor class go?